src/quickstart.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -euo pipefail
+
+OWNER=microshift-io
```

Comment:
Let's add option to override these?
```suggestion
OWNER=${OWNER:-microshift-io}
```